zqd/space/space.go

```diff
@@ -130,6 +119,12 @@ func (m *Manager) Create(name, dataPath string) (*api.SpacePostResponse, error)
 		return nil, err
 	}
 
+	if _, exists := m.spaces[name]; exists {
+		m.mapLock.Unlock()
```

Comment:
the defer will already unlock the mutex, so this isn't needed

Reply:
whoops